ipatests/test_xmlrpc/test_host_plugin.py

```diff
@@ -337,6 +341,84 @@ def test_add_host_with_null_password(self, host):
         host.track_create()
         host.check_create(result)
 
+    @staticmethod
+    def modify_host_config(host_tracker, value):
+        command = host_tracker.make_command(
+            'config_mod',
+            **dict(
+                setattr=u'maxhostnamelen={}'.format(value)))
```

Comment:
When a test is using `make_command` to call one of the `ipa * commands`, it has to use the internal name for the options. You can find them with `ipa show-mappings config-mod` and you will see that `ipa config-mod --maxhostname` is internally using `ipamaxhostnamelength`:
```
# ipa show-mappings config-mod
Parameter                : LDAP attribute
=========                : ==============
maxusername              : ipamaxusernamelength?
maxhostname              : ipamaxhostnamelength?
homedirectory            : ipahomesrootdir?
[...]
```

You need to replace `setattr=u'maxhostnamelen={}'` with `setattr=u'ipamaxhostnamelength={}'`

Reply:
ACK